fs2/src/test/scala/io/finch/fs2/Fs2StreamingSpec.scala

```diff
@@ -1,16 +1,25 @@
 package io.finch.fs2
 
 import _root_.fs2.Stream
-import cats.effect.IO
+import cats.effect.{ConcurrentEffect, Effect, IO}
 import com.twitter.io.Buf
 import io.finch._
 import org.scalatest.prop.GeneratorDrivenPropertyChecks
 
 class Fs2StreamingSpec extends FinchSpec with GeneratorDrivenPropertyChecks {
 
-  checkAll("fs2.streamBody", StreamingLaws[Stream, IO](
-    list => Stream(list:_*),
-    _.map(array => Buf.ByteArray.Owned(array)).compile.toList.unsafeRunSync()
-  ).all)
+  checkEffect[IO]
+  checkConcurrentEffect[IO]
```

Comment:
So how this compiles? I think we need instances of `ContextSwitch[IO]` and `Timer[IO]` to summon `ConcurrentEffect[IO]`?

Reply:
Hmm, there is a `ContextShift` instance in `MissingInstances` :thinking: 
It uses the global execution context.

Reply:
Oh you're right. Ignore me.